JS-Algorithms/challenges.js

```diff
@@ -14,6 +14,11 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 
 const readableTime = (seconds) => {
   // YOUR CODE HERE...
+  const hours = Math.floor(seconds / 3600);
+  const minutes = Math.floor((seconds % 3600) / 60);
+  const remainingSeconds = seconds % 60;
```

Comment:
I recommend not having hardcoded numbers to make the code more understandable, for example:
```suggestion
  const SECONDS_IN_HOUR = 3600
  const SECONDS_IN_MINUTE = 60

  const hours = Math.floor(seconds / SECONDS_IN_HOUR);
  const minutes = Math.floor((seconds % SECONDS_IN_HOUR) / SECONDS_IN_MINUTE);
  const remainingSeconds = seconds % SECONDS_IN_MINUTE;
```

Reply:
Thank you for the recommendation.